fuse/server.go

```diff
@@ -560,13 +552,13 @@ func (ms *Server) EntryNotify(parent uint64, name string) Status {
 
 // SupportsVersion returns true if the kernel supports the given
 // protocol version or newer.
-func (in *InitIn) SupportsVersion(maj, min uint32) bool {
+func (in InitIn) SupportsVersion(maj, min uint32) bool {
```

Comment:
Because `KernelSettings()` returns a non-pointer. The zipfs and unionfs tests don't build right now on Darwin; do they build and work on Linux?